geometry_msgs/Pose pose
---
string asset_id
